src/runtime_src/core/include/ert.h

```diff
@@ -180,7 +180,7 @@ struct ert_start_copybo_cmd {
   uint32_t state:4;          /* [3-0], must be ERT_CMD_STATE_NEW */
   uint32_t unused:6;         /* [9-4] */
   uint32_t extra_cu_masks:2; /* [11-10], = 3 */
-  uint32_t count:11;         /* [22-12], = 15, exclude 'arg' */
+  uint32_t count:12;         /* [22-12], = 16, exclude 'arg' */
```

Comment:
huh?  4+6+2+12+5+4 = 33  
count has to match ert_packet, I am not following this change?

Reply:
sorry, reverted back.